sdk/identity/ci.yml

```diff
@@ -73,16 +73,28 @@ extends:
     LiveTestStages:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
+          PreTestRunSteps:
+            - task: AzureCLI@2
```

Comment:
Could you please add a comment as to why this is here so we know why and if you have a follow-up work item please reference it as well.